CoffeeChat/Controller/NoMatchViewController.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  NoMatchViewController.swift
+//  CoffeeChat
+//
+//  Created by Lucy Xu on 2/29/20.
+//  Copyright © 2020 cuappdev. All rights reserved.
+//
+ import UIKit
+
+ class NoMatchViewController: UIViewController {
+
+     // MARK: - Private View Vars
+    private let availabilityButton = UIButton()
+    private let noMatchLabel = UILabel()
+    private let noMatchTitleLabel = UILabel()
+    private let surprisedPearImage = UIImageView()
+
+     // MARK: - Private Constants
+    private let buttonSize = CGSize(width: 225, height: 54)
+    private let buttonBottomPadding: CGFloat = LayoutHelper.shared.getCustomVerticalPadding(size: 102)
+    private let imageBottomPadding: CGFloat = LayoutHelper.shared.getCustomVerticalPadding(size: 36)
+    private let imageWidth = (UIScreen.main.bounds.width / 375) * 176
+    private let subtitleLabelPadding: CGFloat = LayoutHelper.shared.getCustomVerticalPadding(size: 24)
+    private let titleLabelPadding: CGFloat = LayoutHelper.shared.getCustomVerticalPadding(size: 92)
```

Comment:
We should just put these in the function where they're used, in the `viewDidLoad`

Reply:
oh yes you right